src/routes/Routes.tsx

```diff
@@ -185,6 +187,16 @@ const Routes = () => {
             hasSubscribedColonies: false,
           })}
         />
+        <AlwaysAccesibleRoute
```

Comment:
Do you mean this should use the WalletRequiredRoute or to provide a check in the decision preview page to check if the user is logged in? I did look into using WalletRequiredRoute but routeProps work a bit different it may need some modifications to work in this case

Reply:
I think you should use `WalletRequiredRoute` and adopt the props accordingly.

Reply:
> I think you should use `WalletRequiredRoute` and adopt the props accordingly. DM me if you need more details about it.

It's actually not that easy. I think it will require modifying `WalletRequiredRoute` or its props. When I tried quick and dirty, it crashes the app so I would need more time to investigate. I think it's better to leave as is and have a separate issue to add some restrictions to the route. I'll create it.